index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML forms</title>
+<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="blocks">
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset>
+       <legend>Personal information</legend>
+
+       <div class="form-field">
+      <label for="surname">Surname:</label>
+        <input
+        id="surname"
+        type="text"
+        name="surname"
+        required
+        autocomplete="off"
+        >
+      </div>
+
+      <div class="form-field">
+        <label for="name">Name:</label>
+        <input
+        id="name"
+        type="text"
+        name="name"
+        required
+        autocomplete="off"
+        >
```

Comment:
```suggestion
        <input
          id="name"
          type="text"
          name="name"
          required
          autocomplete="off"
        >
```
Добавь отступы атрибутам